scripts/test/nightly/runTests.sh

```diff
@@ -69,27 +67,56 @@ set +e
 
 END_EXIT_CODE=0
 
+docker buildx rm sleeper
+docker buildx create --name sleeper --use
+
+copyFolderForParallelRun() {
+    set -e
```

Comment:
With `set -e` here, if this fails, I don't think we'll get a record of this failure. I think it'll look like the test suite passed, when actually it didn't run these tests. In the results, the only way we know that a test suite exists at all is the file `$OUTPUT_DIR/$TEST_NAME.status`, which gets created at the end of `runMavenSystemTests`.